appIos/appIos.xcodeproj/project.pbxproj

```diff
@@ -324,7 +324,7 @@
 				ENABLE_PREVIEWS = YES;
 				FRAMEWORK_SEARCH_PATHS = "";
 				INFOPLIST_FILE = appIos/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.4;
```

Comment:
The `searchable` functionality seems to be a iOS 15 only thing. Xcode got very annoyed without this